internal/pkg/template/templates/workloads/jobs/scheduled-job/manifest.yml

```diff
@@ -46,7 +46,21 @@ platform: {{.Platform.PlatformString}}   # See https://aws.github.io/copilot-cli
 #secrets:                      # Pass secrets from AWS Systems Manager (SSM) Parameter Store.
 #  GITHUB_TOKEN: GITHUB_TOKEN  # The key is the name of the environment variable, the value is the name of the SSM parameter.
 
+{{- if not .Environments}}
```

Comment:
I wonder if we should make this more specific, in case we add other config to `.Environments`? Or we could change it later.

Reply:
I will this one to be more specific.

Reply:
_Correct if am wrong:_

Thinking about it more, `{{- if not .Environments}}` makes sense because we have `Environments : map[string]*LoadBalancedWebServiceConfig`. So, whenever we have `Environments` is empty, we generate the default manifest file, the same as before, which supports backward compatibility.

If we have to add any other config just like `Placement` in future, we can add those configs in the `else` case when the Environments is not empty.